HWLesson 1.4/HWLesson 1.4/Program.cs

```diff
@@ -0,0 +1,111 @@
+﻿
+namespace HWLesson14
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter count of elements!");
+            var countElements = Console.ReadLine();
+            if (int.TryParse(countElements, out int count) && count > 0)
+            {
+                int[] massiveElem = new int[count];
+
+                InputRandomElem(massiveElem);
+
+                string[] evenNumber = new string[CountEven(massiveElem)];
+                string[] oddNumber = new string[massiveElem.Length - CountEven(massiveElem)];
+                int evenIndex = 0;
+                int oddIndex = 0;
+
+                foreach (var i in massiveElem)
+                {
+                    if (i % 2 == 0)
+                    {
+                        evenNumber[evenIndex] = IntToEngAlphabet(i);
+                        evenIndex++;
+                    }
+                    else
+                    {
+                        oddNumber[oddIndex] = IntToEngAlphabet(i);
+                        oddIndex++;
+                    }
```

Comment:
This logic can be moved to a separate method

Reply:
i see